src/js/components/Tabs/Tabs.js

```diff
@@ -27,8 +40,13 @@ const Tabs = forwardRef(
     const { format } = useContext(MessageContext);
     const { activeIndex: propsActiveIndex, onActive } = rest;
     const [activeIndex, setActiveIndex] = useState(rest.activeIndex || 0);
+    const [focusIndex, setFocusIndex] = useState(activeIndex);
     const [activeContent, setActiveContent] = useState();
     const [activeTitle, setActiveTitle] = useState();
+    const [disableLeftArrow, setDisableLeftArrow] = useState();
+    const [disableRightArrow, setDisableRightArrow] = useState();
+    const [overflow, setOverflow] = useState(undefined);
+    const targetRef = useRef();
```

Comment:
Can we add a comment of why this is called `targetRef`?

Reply:
I'm thinking of changing the name to `headerRef` to make it more clear that this ref is for the tabs header container

Reply:
Changed name to `headerRef` in latest push